tests/c/test-ext-3.c

```diff
@@ -14,32 +14,51 @@ extern unsigned nd_uint(void);
 
 int main(void) {
 
-  // TODO: Use sea_dsa_new();
-
   {
     // Test multiple attributes set
-    int *p;
+    void *p = sea_dsa_new();
```

Comment:
I am not sure that this adds much over `noalias` attribute (i.e., `malloc` attribute in C). 
You can just mark that function but no need to have any special processing in `DsaLocal`. It will be processed as an external function that behaves like `malloc`